packages/fluentui/docs/src/prototypes/VirtualizedStickyTree/itemsGenerator.ts

```diff
@@ -0,0 +1,24 @@
+import * as _ from 'lodash';
+
+function getItems(minItems = 5, maxItems = 15, maxLevel = 2, maxSticky = 10) {
+  function getItemsNumber(min, max) {
```

Comment:
Propose add typings.

Reply:
Thank you for the review, the typings are add in this [commit](https://github.com/microsoft/fluentui/pull/15461/commits/69e840d0ab7ed4db265d4d288b67dcb06aad0b6e)